tests/test_exchange_rate.py

```diff
@@ -3,6 +3,15 @@
 import pytest
 from click import ClickException
 import requests_mock as rm
+import requests
+
+
+def _request_error_throwing_test(requests_mock, expected_error):
+    with pytest.raises(ClickException):
+        requests_mock.get(rm.ANY,
+                          exc=expected_error)
+        day = date(2020, 5, 22)
+        ExchangeRate("USD", day, day)

```

Comment:
What about also testing the error message as well. You could pass it as a parameter.

Reply:
Fixed all the tests in this file to assert that the error message is correct. Also removed some tests that were repeated